test-common/src/main/java/com/gentics/mesh/test/docker/MeshContainer.java

```diff
@@ -50,6 +53,12 @@
  */
 public class MeshContainer extends GenericContainer<MeshContainer> {
 
+	private static final String PATH_UPLOADS = "/uploads";
```

Comment:
The changes below allow overriding container's bind folders to simulate inaccessible/limited folders required for Mesh to operate. The tests utilizing this function will be added shortly.